MLS.Agent.Tools.Tests/MLS.Agent.Tools.Tests.csproj

```diff
@@ -29,8 +29,15 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="FSharp.Core" />
+    <PackageReference Include="FSharp.Core" Version="5.0.2" />
```

Comment:
Don't specify F# package versions; they're all handled in the root `Directory.Build.targets`.